lib/internal/structured_clone.js

```diff
@@ -6,13 +6,13 @@ const {
 } = require('internal/worker/io');
 
 let channel;
-function structuredClone(value, transfer) {
+function structuredClone(value, options) {
```

Comment:
Unsure how browsers will implement this, and if it matters at all, but we should probably mark option as optional so `structuredClone.length === 1` is true.
```suggestion
function structuredClone(value, options = undefined) {
```